homeassistant/components/niko_home_control/light.py

```diff
@@ -36,9 +42,55 @@ async def async_setup_platform(
     config: ConfigType,
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the light platform."""
+
+    if config.get("platform") == DOMAIN:
+        if not hass.config_entries.async_entries(DOMAIN):
```

Comment:
this if statement can be removed

Reply:
Are you sure? I think that will result in the import running all the time no?

Reply:
Ah ok, I see, nvm :D